02-cpu-configure/cpu-configure.sh

```diff
@@ -106,4 +106,4 @@ chmod +x /usr/local/bin/pcie-list-slots
 # Linux kernel will ignore the suid bit.
 cd "$INIT_PWD"
 make install
-chmod u+s /usr/local/bin/pcie-rescan /usr/local/bin/pcie-remove /usr/local/bin/ioc-restart
+make install-setuid
```

Comment:
Just execute `make instal-setuid`, it will execute the `install` rule which it depends.